libs/core/src/components/pds-tabs/stories/pds-tabs.docs.mdx

```diff
@@ -1,19 +1,11 @@
-import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
-import { html, render } from 'lit-html';
+import { Meta, Story, Canvas, ArgTypes } from '@storybook/blocks';
+import { html, render } from 'lit';
 
 import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
 
 import * as stories from './pds-tabs.stories.js';
 
-<Meta
-  title="components/Tabs"
-  component="pds-tabs"
-  argTypes={extractArgTypes('pds-tabs')}
-  subcomponents={{
-    PdsTab: 'pds-tab',
-    PdsTabPanel: 'pds-tabpanel',
```

Comment:
I did notice the subcomponent tabs are no longer showing up on the tabs documentation. Is this intentional or due to upgrading? Not a blocker, but would be great to have those if possible.

Reply:
Yes, @pixelflips there is some funky behavior with subcomponents in the latest version. Let me see what I can dig back up.

Reply:
Ok cool, can def be handled in a follow-up if needed. Thanks for checking!

Reply:
Here is the information that I obtained when I was going through each of the components and ran into `subcomponents`.

https://github.com/storybookjs/storybook/issues/20782#issuecomment-1482771013

Reply:
Thanks for sharing! I have also used the subcomponents on the sortable, so I will check out the info, but if you have any suggestions let me know!